src/test/java/net/pms/configuration/FormatConfigurationRegressionTest0.java

```diff
@@ -10,966 +10,707 @@
 
     // Regression assertion (captures the current behavior of the code)
     org.junit.Assert.assertTrue("'" + str0 + "' != '" + "rm"+ "'", str0.equals("rm"));
-
   }
 
   @Test
   public void test73() throws Throwable {
-
-    if (debug) { System.out.format("%n%s%n","RegressionTest0.test73"); }
-
     java.lang.String str0 = net.pms.configuration.FormatConfiguration.WAVPACK;
 
     // Regression assertion (captures the current behavior of the code)
     org.junit.Assert.assertTrue("'" + str0 + "' != '" + "wavpack"+ "'", str0.equals("wavpack"));
-
   }
 
   @Test
   public void test74() throws Throwable {
-
-    if (debug) { System.out.format("%n%s%n","RegressionTest0.test74"); }
-
     java.lang.String str0 = net.pms.configuration.FormatConfiguration.MOV;
 
     // Regression assertion (captures the current behavior of the code)
     org.junit.Assert.assertTrue("'" + str0 + "' != '" + "mov"+ "'", str0.equals("mov"));
-
   }
 
   @Test
   public void test75() throws Throwable {
-
-    if (debug) { System.out.format("%n%s%n","RegressionTest0.test75"); }
-
     java.lang.String str0 = net.pms.configuration.FormatConfiguration.SORENSON;
 
     // Regression assertion (captures the current behavior of the code)
     org.junit.Assert.assertTrue("'" + str0 + "' != '" + "sor"+ "'", str0.equals("sor"));
-
   }
 
   @Test
   public void test76() throws Throwable {
-
-    if (debug) { System.out.format("%n%s%n","RegressionTest0.test76"); }
-
     java.lang.String str0 = net.pms.configuration.FormatConfiguration.MLP;
 
     // Regression assertion (captures the current behavior of the code)
     org.junit.Assert.assertTrue("'" + str0 + "' != '" + "mlp"+ "'", str0.equals("mlp"));
-
   }
 
   @Test
   public void test77() throws Throwable {
-
-    if (debug) { System.out.format("%n%s%n","RegressionTest0.test77"); }
-
     java.lang.String str0 = net.pms.configuration.FormatConfiguration.AMR;
 
     // Regression assertion (captures the current behavior of the code)
     org.junit.Assert.assertTrue("'" + str0 + "' != '" + "amr"+ "'", str0.equals("amr"));
-
   }
 
   @Test
   public void test78() throws Throwable {
-
-    if (debug) { System.out.format("%n%s%n","RegressionTest0.test78"); }
-
     java.lang.String str0 = net.pms.configuration.FormatConfiguration.MP3;
 
     // Regression assertion (captures the current behavior of the code)
     org.junit.Assert.assertTrue("'" + str0 + "' != '" + "mp3"+ "'", str0.equals("mp3"));
-
   }
 
   @Test
   public void test79() throws Throwable {
-
-    if (debug) { System.out.format("%n%s%n","RegressionTest0.test79"); }
-
     java.lang.String str0 = net.pms.configuration.FormatConfiguration.DTSHD;
 
     // Regression assertion (captures the current behavior of the code)
     org.junit.Assert.assertTrue("'" + str0 + "' != '" + "dtshd"+ "'", str0.equals("dtshd"));
-
   }
 
   @Test
   public void test80() throws Throwable {
-
-    if (debug) { System.out.format("%n%s%n","RegressionTest0.test80"); }
-
```

Comment:
@Sami32 Standalone, but I'm not sure I'm going to use it much more. "Cleaning up" the tests it makes and make it into something useful is as much job as writing them from scratch, the last time I used it I regretted doing so.

[Randoop](https://randoop.github.io/randoop/)